packages/typespec-test/test/ai/generated/typespec-ts/review/ai-client.api.md

```diff
@@ -646,6 +646,9 @@ export interface FileContentResponse {
     content: Uint8Array;
 }
 
+// @public
+export type FileContents = string | NodeJS.ReadableStream | ReadableStream<Uint8Array> | Uint8Array | Blob;
```

Comment:
just curious why this is added? is this expected?

Reply:
Seems like this spec has an anonymous model for MFD which isn't properly supported yet. `FileContents` should be used in `uploadFile` but is currently `Uint8Array`.

Reply:
Will address anonymous models in a follow up issue